### PR TITLE
addErrataUpdate API for multiple servers

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

Create a new API endpoint for addErrataUpdate, that take multiple servers as argument. 
## GUI diff

No difference.

- [x] **DONE**

## Documentation
API Doc
- [x] **DONE**

## Test coverage
- Unit tests were added

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/17895

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests" 
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/code/src/com/redhat/rhn/frontend/xmlrpc/chain/ActionChainHandler.java [hunk 1]
```diff
@@ -293,6 +293,53 @@ public Integer addErrataUpdate(User loggedInUser,
...
+     * @apidoc.param #session_key()
+     * @apidoc.param #array_single_desc("int", "sids", "System IDs")
+     * @apidoc.param #array_single_desc("int", "errataIds", "Errata ID")
+     * @apidoc.param #param_desc("string", "chainLabel", "Label of the chain")
+     * @apidoc.returntype #param_desc("int", "actionId", "The action id of the scheduled action")
+     */
+    public Integer addErrataUpdate(User loggedInUser,
+                                    List<Integer> sids,
+                                    List<Integer> errataIds,
+                                    String chainLabel) {
```
Comment (mcalmer) [inline java/code/src/com/redhat/rhn/frontend/xmlrpc/chain/ActionChainHandler.java:315]: Can we change the "single" version from line 269 to call this one ?
We have a lot of duplicate code.

**Review by mcalmer — CHANGES_REQUESTED** [review]
In the Handler we need both, the old and the new signature.
As `ActionChainManager` is internal, we can just change all the code which is calling it.
Seems that only the Handler and tests are calling it. So here it is ok, to just change it.

**Review by mcalmer — APPROVED** [review]
Looks good. Thanks

**Comment by mbussolotto** [comment]
ci tests: https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/1392/

**Comment by mbussolotto** [comment]
new CI tests: https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/1394/

**Comment by mbussolotto** [comment]
another CI test: https://ci.suse.de/view/Manager/view/Uyuni-PRs/job/uyuni-prs-ci-tests/1414/

**Pull request merged**